bedrock/foundation/templates/foundation/mocosc.html

```diff
@@ -45,7 +45,7 @@ <h1 class="page-title">{{ _('About the Mozilla Corporation Steering Committee')
     <li><a href="https://blog.mozilla.org/press/bios/david-bryant/">David Bryant, {{ _('Interim CTO and VP, Platform Engineering') }}</a></li>
     <li><a href="https://blog.mozilla.org/press/bios/jim-cook/">Jim Cook, {{ _('Chief Finance Officer') }}</a></li>
     <li><a href="https://blog.mozilla.org/press/bios/denelle-dixon-thayer/">Denelle Dixon-Thayer, {{ _('Chief Legal and Business Officer') }}</a></li>
-    <li><a href="https://blog.mozilla.org/press/bios/david-slater/">David Slater, {{ _('Chief of Staff and SVP of Operations') }}</a></li>
     <li><a href="https://blog.mozilla.org/press/bios/mark-mayo/">Mark Mayo, {{_('Senior Vice President, Firefox')}}</a></li>
+    <li><a href="https://blog.mozilla.org/press/bios/david-slater/">David Slater, {{ _('Chief of Staff and SVP of Operations') }}</a></li>
```

Comment:
I was wondering about the order and just now noticed it is alphabetical based on *last* name ;)